osconfig_tests/utils/utils.go

```diff
@@ -111,3 +114,56 @@ func GetStatusFromError(err error) string {
 	}
 	return fmt.Sprintf("%v", err)
 }
+
+// CreateTestInstance is an utility function to create gce instance
+func CreateTestInstance(metadataitems []*api.MetadataItems, client daisyCompute.Client, machineType, image, name, projectID, zone, serviceAccountEmail string, serviceAccountScopes []string) (*compute.Instance, error) {
```

Comment:
Consider renaming to `CreateComputeInstance` or `CreateGceInstance` to obviate the comment.

Reply:
done